src/components/Pun/Pun.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+
+const Pun = ({date,initialVotes,dev,context,message}) => {
+
+  const [votes, setVotes] = useState(initialVotes)
+  return (
+    <tr>
+      <td>{date}</td>
+      <td>{votes}</td>
+      <td>{dev}</td>
+      <td>{context}</td>
+      <td>{message}</td>
+      <td>
+        <button onClick={()=>{setVotes(votes => votes+=1)}}>Votar</button>
```

Comment:
You can make a function and use it here. It's going to be easier to test the behavior and make sure the function is being called when the button is clicked. You can have a `utils` to store this function and use it here.

Reply:
If you have time, its nice create an component only for this button.
VoteButton Folder with VoteButton.js and VoteButton.test.js for example